acestep/models/common/dcw_correction.py

```
@@ -0,0 +1,257 @@
+    return _bct_to_btc(x_new[:, :, :out_T]).to(dtype=x.dtype)
+
+
+def dcw_high(x: torch.Tensor, y: torch.Tensor, scaler: float, wavelet: str = "haar") -> torch.Tensor:
+    """Apply differential correction to the high-frequency sub-band only."""
+    if scaler == 0.0:
+        return x
+    pair = _dwt_pair(x, y, wavelet)
+    if pair is None:
+        return x
+    xl, xh, _yl, yh, iwt, out_T = pair
+    xh_new = [xhi + scaler * (xhi - yhi) for xhi, yhi in zip(xh, yh)]
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)